propolis/src/hw/nvme/cmds.rs

```diff
@@ -589,8 +590,13 @@ impl PrpIter<'_> {
         assert!(self.remain > 0);
         assert!(self.error.is_none());
 
+        // PRP Entry Layout
+        // | 63                                      n + 1 | n . . . . . . . 2 1 0 |
+        // |         page base address                     |      offset     | 0 0 |
```

Comment:
```suggestion
        // | 63                                      n + 1 | n . . . . . . 2 | 1 0 |
        // |         page base address                     |      offset     | 0 0 |
```

Small nit: just lining it up

Reply:
suggestion done in eb141578c3763d1e1ed035fe307159b20ae8e5fe